examples/simple-hive-cluster.yaml

```diff
@@ -4,7 +4,7 @@ kind: HiveCluster
 metadata:
   name: simple-hive-derby
 spec:
-  version: 2.3.9-stackable0.4.0
+  version: 3.1.3-stackable0.1.0
```

Comment:
bitte raus nehmen

Reply:
Changed it in this PR: https://github.com/stackabletech/hive-operator/pull/211/files